server/embed/etcd.go

```diff
@@ -744,6 +744,7 @@ func (e *Etcd) serveClients() (err error) {
 	etcdhttp.HandleVersion(mux, e.Server)
 	etcdhttp.HandleMetrics(mux)
 	etcdhttp.HandleHealth(e.cfg.logger, mux, e.Server)
```

Comment:
what about the old healthz code?

Reply:
I am keeping the old health endpoint, because some checks are not going to be included in the new healthz, such as all alarms, and it uses a different set of query params like "serializable=true".

Reply:
Oh, I would find having old "/health" and "/healthz" confusing. Can we skip "/healtz" in this PR?